kclvm/parser/fuzz/fuzz_targets/fuzz_parser.rs

```diff
@@ -0,0 +1,35 @@
+#![no_main]
+use kclvm::PanicInfo;
+use kclvm_parser::parse_expr;
+use libfuzzer_sys::arbitrary::Arbitrary;
+use libfuzzer_sys::fuzz_target;
+use serde_json;
+use std::panic::catch_unwind;
+use std::panic::set_hook;
+
+#[derive(Arbitrary, Debug)]
+enum ParserMethod<'a> {
+    ParseExpr { src: &'a str },
+}
+
+fuzz_target!(|method: ParserMethod| {
```

Comment:
You can put fuzz tests in the kclvm/tests folder, which is divided into parser fuzz tests.

Reply:
Add a fast access to fuzz test in makefile?